engine/tests/raycast.cpp

```diff
@@ -0,0 +1,130 @@
+#include <doctest/doctest.h>
+
+#include <cubos/core/ecs/name.hpp>
+
+#include <cubos/engine/collisions/plugin.hpp>
+#include <cubos/engine/collisions/raycast.hpp>
+#include <cubos/engine/fixed_step/plugin.hpp>
+#include <cubos/engine/prelude.hpp>
+#include <cubos/engine/transform/local_to_world.hpp>
+#include <cubos/engine/transform/plugin.hpp>
+#include <cubos/engine/transform/position.hpp>
+
+#include "utils.hpp"
+
+using cubos::core::ecs::Name;
+using cubos::core::ecs::World;
+using namespace cubos::engine;
+
+TEST_CASE("simple use cases of raycast")
```

Comment:
```suggestion
TEST_CASE("cubos::engine::Raycast")
```

for consistency with other tests :relieved: